servers/rendering/renderer_rd/storage_rd/render_scene_buffers_rd.cpp

```diff
@@ -161,6 +174,32 @@ void RenderSceneBuffersRD::configure(RID p_render_target, const Size2i p_interna
 		create_texture(RB_SCOPE_BUFFERS, RB_TEX_DEPTH, format, usage_bits);
 	}
 
+	// Create our MSAA buffers
+	if (msaa_3d == RS::VIEWPORT_MSAA_DISABLED) {
+		texture_samples = RD::TEXTURE_SAMPLES_1;
+	} else {
+		RD::DataFormat format = base_data_format;
+		uint32_t usage_bits = RD::TEXTURE_USAGE_COLOR_ATTACHMENT_BIT | RD::TEXTURE_USAGE_CAN_COPY_FROM_BIT | RD::TEXTURE_USAGE_SAMPLING_BIT;
+
+		const RD::TextureSamples ts[RS::VIEWPORT_MSAA_MAX] = {
+			RD::TEXTURE_SAMPLES_1,
+			RD::TEXTURE_SAMPLES_2,
+			RD::TEXTURE_SAMPLES_4,
+			RD::TEXTURE_SAMPLES_8,
+		};
+
+		texture_samples = ts[msaa_3d];
+
+		create_texture(RB_SCOPE_BUFFERS, RB_TEX_COLOR_MSAA, format, usage_bits, texture_samples);
+
+		// TODO: the only difference between mobile and forward+ here is that we don't check TEXTURE_USAGE_SAMPLING_BIT in forward+ even though we do set
+		// this may be a bug, or may be something we need to further test.
```

Comment:
I looked into this. Forward plus was manually specifying ``RD::TEXTURE_USAGE_DEPTH_STENCIL_ATTACHMENT_BIT | RD::TEXTURE_USAGE_SAMPLING_BIT`` Since ``TEXTURE_USAGE_CAN_COPY_FROM_BIT`` is not checked by ``texture_is_format_supported_for_usage``, the result is functionally identical. 

Accordingly, you can remove this comment

Reply:
Sweet, so the only decision left here is the discussion we had around the use of D24S8.
It seems we should prefer D32S8 always?